diagnostic.md

```diff
@@ -57,7 +87,7 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$("li").hide()
```

Comment:
So this would hide the list item. We have to traverse up the DOM to reach the body.

`$('li').parent().parent().hide()`